src/config.c

```diff
@@ -448,14 +448,14 @@ static size_t find_white(const struct cfg_string *str, size_t current, enum cfg_
 }
 */
 /* find white space and cut off with a null char */
-static void rstrip_inplace(char *str)
+static void rstrip_inplace(char *str, size_t len)
```

Comment:
valgrind was occasionally logging invalid reads from the line len=strlen(str), hence now passing the length of the string that's already known